### PR TITLE
complete setup of dbs and seeding, write test and function for get to…

### PR DESCRIPTION
**Pull request by Holquaria**
…pics

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+beforeEach(() => seed(testData))
+
+describe('GET /api/topics', () => {
+    test('should return an array of all topic objects', () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics).toBeInstanceOf(Array);
```
Comment (roisinoreilly) [inline __tests__/app.test.js]: Do we need this line if your forEach is passing the test?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          topics.forEach((topic) => {
+            expect(topic).toEqual(
+              expect.objectContaining({
+                slug: expect.any(String),
+                description: expect.any(String),
+              })
```
Comment (roisinoreilly) [inline __tests__/app.test.js]: We need to check if the array is empty so that we don't get a false positive for this test, can we test the length of this array is what you're expecting

File: controllers/errors.controllers.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+exports.handle500Error = (err, req, res, next) => {
```
Comment (roisinoreilly) [inline controllers/errors.controllers.js:5]: Great job including this

**Review by roisinoreilly — CHANGES_REQUESTED** [review]
Great work! Could you please separate your PRs for ticket 2 and 3 (so the readme is it's own request), your commit messages should never use the word "and" - this signifies they should be separate commits. 

Can you also change your test to make sure that your array isn't empty and have a think about what would happen if it was.

File: __tests__/app.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("should give a custom message for an invalid path", () => {
+    return request(app)
+      .get("/api/tobiics")
```
Comment (Pipe-Smoking-Rabbit) [inline __tests__/app.test.js:30]: Just so you're aware this bad path error only needs to be tested for once as you have done above - you don't have to worry about putting it into every endpoint test block

**Review by Pipe-Smoking-Rabbit — APPROVED** [review]
Looks great, feel free to merge!
